wolftpm/visibility.h

```diff
@@ -31,7 +31,8 @@
 */
 
 #if defined(BUILDING_WOLFTPM)
-    #if defined(_MSC_VER) || defined(__CYGWIN__)
+    #if defined(_MSC_VER) || defined(__CYGWIN__) || \
+      defined(__MINGW32__) || defined(__MINGW64__)
```

Comment:
Can you add MinGW build instructions to `README.md`?

For ref here is what you sent me:

```
cd wolfssl
export PREFIX=$PWD/../tmp
configure --host=i686 CC=i686-w64-mingw32-gcc CFLAGS="-DWIN32 -DMINGW -D_WIN32_WINNT=0x0600" LIBS="-lws2_32" --prefix=$PREFIX
make
make install
cd ../wolftpm/
./configure --host=i686 CC=i686-w64-mingw32-gcc CFLAGS="-DWIN32 -DMINGW -D_WIN32_WINNT=0x0600" LIBS="-lws2_32 -ltbs" --prefix=$PREFIX --enable-winapi
make
```

Reply:
I added the winapi specific instructions in `docs/WindowTBS.md` https://github.com/wolfSSL/wolfTPM/pull/127/files/62ff638dc7937d94f19e2ebf96e4b769fcf59113#diff-ff9c2ef8f041e518689e41a95875e1ec7789ae59d7c4bf0560a33df554f99759R41-R52

Do also want those in the top level `README.md`?

Reply:
Nope, that is perfect. I missed that!